tests/stats_test.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import pandas as pd
+from base_test import ArkoudaTest
+from context import arkouda as ak
+
+
+class StatsTest(ArkoudaTest):
+    def setUp(self):
+        ArkoudaTest.setUp(self)
+        self.x = ak.arange(10, 20)
+        self.pdx = pd.Series(np.arange(10, 20))
+        self.y = ak.array([2, 1, 4, 5, 8, 12, 18, 25, 96, 48])
+        self.pdy = pd.Series(np.array([2, 1, 4, 5, 8, 12, 18, 25, 96, 48]))
+        self.u = ak.cast(self.y, ak.uint64)
+        self.pdu = pd.Series(np.array([2, 1, 4, 5, 8, 12, 18, 25, 96, 48], dtype=np.uint64))
+        self.b = ak.arange(10) % 2 == 0
+        self.pdb = pd.Series(np.arange(10) % 2 == 0)
+        self.f = ak.array([i**2 for i in range(10)], dtype=ak.float64)
+        self.pdf = pd.Series(np.array([i**2 for i in range(10)], dtype=np.float64))
+        self.s = ak.array([f"string {i}" for i in range(10)])
+        self.c = ak.Categorical(self.s)
+
+    def test_mean(self):
+        self.assertEqual(self.x.mean(), self.pdx.mean())
+        self.assertEqual(self.y.mean(), self.pdy.mean())
+        self.assertEqual(self.u.mean(), self.pdu.mean())
+        self.assertEqual(self.b.mean(), self.pdb.mean())
+        self.assertEqual(self.f.mean(), self.pdf.mean())
+
+    def test_var(self):
+        # Note the numpy and pandas var/std differ, we follow numpy by default
+        # The pandas version requires ddof = 1
```

Comment:
Can we add this as a `Note` to the `var` method docstring? I am just worried this will get missed otherwise.

Reply:
it's kinda already there. This is saying that 0 is the maximum likelihood estimate and 1 is the unbiased estimator. It might be worthwhile to give a more explicit call out that numpy uses maximum likelihood estimate by default and pandas uses unbiased estimator. and that we default to numpy
https://github.com/Bears-R-Us/arkouda/blob/c8ce6c449562ab6ce67363a347043f1b8d86a76b/arkouda/pdarrayclass.py#L2005-2015

Reply:
Yeah you are right. I only looked at the docstring of the `var` function inside of the class, not the file level function.